### PR TITLE
For Chunker Issue252

### PR DESCRIPTION
**Pull request by qiangning**

### CODE REVIEW

**Comment by bhargav** [comment]
Can't set status; build succeeded.

**Comment by qiangning** [comment]
Why does  illinois-ner fail to build? That's weird since I am not working on that.
```
[INFO] Reactor Summary:
[INFO] 
[INFO] illinois-cogcomp-nlp ............................... SUCCESS [  0.025 s]
[INFO] illinois-core-utilities ............................ SUCCESS [  2.555 s]
[INFO] LBJava-NLP-tools ................................... SUCCESS [  0.200 s]
[INFO] illinois-curator ................................... SUCCESS [  0.170 s]
[INFO] illinois-tokenizer ................................. SUCCESS [  0.121 s]
[INFO] illinois-corpusreaders ............................. SUCCESS [  0.281 s]
[INFO] illinois-edison .................................... SUCCESS [  3.995 s]
[INFO] illinois-lemmatizer ................................ SUCCESS [  0.474 s]
[INFO] illinois-pos ....................................... SUCCESS [  0.203 s]
[INFO] illinois-chunker ................................... SUCCESS [  8.740 s]
[INFO] illinois-ner ....................................... FAILURE [02:53 min]
[INFO] big-data-utils ..................................... SKIPPED
```

File: chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java [hunk 1]
```diff
@@ -99,8 +180,17 @@ public void writeModelsToDisk(String dir, String modelName){
         System.out.println("Done training, models are in " + dir+File.separator+modelName+".lc (.lex)");
     }
     public static void main(String[] args) {
-        ChunkerTrain trainer = new ChunkerTrain();
-        trainer.trainModels();
-        trainer.writeModelsToDisk();
+        if(args.length!=4&&args.length!=5){
+            System.out.println("Usage1: ...ChunkerTrain traindata modeldir modelname round");
+            System.out.println("Usage2: ...ChunkerTrain traindata modeldir modelname max_round dev_ratio");
```
Comment (mssammon) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: add a System.exit(-1) after the usage output

Comment (mssammon) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: This is a non-deterministic problem with the CI build system: it sometimes fails to download all the NER resource jars and this is the message that results.

File: chunker/README.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ Here is how you can add maven dependencies into your program:
     </repositories>
     
     <dependencies>
+    <!--Remove this dependency if you want to train your own model-->
```
Comment (danyaljj) [inline chunker/README.md]: 👍

File: chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java (hunk 1 above)
Comment (qiangning) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: After restarting semaphore for more than 5 times, it passed finally. Can you accept this pull request?

**Comment by danyaljj** [comment]
Mark seems to be happy. Merging!

**Comment by danyaljj** [comment]
@qiangning pro tip: https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by qiangning** [comment]
@danyaljj , but closing via commit msgs doesn't lead to a merge, does it?

**Comment by danyaljj** [comment]
Not, it closes the issue when you merge the PR.

**Pull request merged**